mantidproject/mantid
# Updated images in Stitch1DMany documentation

**Description of work**

**Purpose of work**
After completing #35466 it became clear that the screenshots in the documentation for Stitch1DMany algorithm were now out of date. The purpose of this work is to make the documentation consistent with the current behaviour of the algorithm.

**Summary of work**
Changes to screenshots in the Stitch1dMany documentation.

**Further detail of work**
Although it was planned to only change the final screenshot for the Stitch1DMany algorithm documentation it became clear that both screenshots were so old it would be more consistent to change both. The documentation now more accurately reflects what users will see when using Mantid.


**To test:**

- remove build/docs/doctrees
- build the documents again
- Check images are different from those currently at https://docs.mantidproject.org/nightly/algorithms/Stitch1DMany-v1.html 


Fixes #35470

*This does not require release notes* because **the changes that necessitated the new screenshots (#35466) already has a release note**


---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
